### PR TITLE
new implementation of modelling plots

### PR DESCRIPTION
**Pull request by stevekogo**

### CODE REVIEW

**Comment by stevekogo** [comment]
@africanmathsinitiative/instat I am not sure if visreg package has been added to the project, otherwise install the package in R to successfully run the project.

**Comment by stevekogo** [comment]
@rdstern I have implemented the plots as you had suggested, however, the layout of the dialogue is still questionable. Have a look at it then give suggestion on how in can be improved.

**Comment by dannyparsons** [comment]
@stevekogo will accept for now so Roger and look and comment on it.

File: instat/static/InstatObject/R/Rsetup.R [hunk 1]
```diff
@@ -44,6 +44,7 @@ library(lazyeval)
 library(stringr)
 library(httr)
 library(jsonlite)
+library(visreg)
```
Comment (dannyparsons) [inline instat/static/InstatObject/R/Rsetup.R]: @africanmathsinitiative/instat still in the process of adding new packages so install this manually for now.

**Pull request merged**